packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.tsx

```diff
@@ -1,35 +1,62 @@
 import React from 'react';
 import { WalletText } from '../../../../components/Base';
-import DerivedMT5Icon from '../../../../public/images/mt5-derived.svg';
+import { WalletListCardBadge } from '../../../../components';
 import { MT5TradeDetailsItem } from './MT5TradeDetailsItem';
 import ImportantIcon from '../../../../public/images/ic-important.svg';
 import useDevice from '../../../../hooks/useDevice';
 import { MT5TradeLink } from './MT5TradeLink';
 import './MT5TradeScreen.scss';
+import { useModal } from '../../../../components/ModalProvider';
+import { MarketTypeToIconMapper, MarketTypeToTitleMapper } from '../../constants';
+import { useActiveWalletAccount, useMT5AccountsList } from '@deriv/api';
 
 const MT5TradeScreen = () => {
     const { isDesktop } = useDevice();
+    const { getModalState } = useModal();
+    const { data } = useMT5AccountsList();
+    const { data: activeWalletData } = useActiveWalletAccount();
+
+    const marketType = getModalState('marketType');
+    const details = data?.filter(account => account.market_type === marketType)[0];
 
     return (
         <div className='wallets-mt5-trade-screen'>
             <div className='wallets-mt5-trade-screen__details'>
                 <div className='wallets-mt5-trade-screen__details-description'>
                     <div className='wallets-mt5-trade-screen__details-description--left'>
-                        <DerivedMT5Icon />
-                        <div className='wallets-mt5-trade-screen__details-description-title'>
-                            <WalletText size='sm'>Derived</WalletText>
+                        {MarketTypeToIconMapper[marketType || 'all']}
+                        <div className='wallets-mt5-trade-screen__label'>
+                            <div className='wallets-mt5-trade-screen__title'>
+                                <WalletText lineHeight='3xs' size='sm'>
+                                    {MarketTypeToTitleMapper[marketType || 'all']}{' '}
+                                    {details?.landing_company_short?.toUpperCase()}
+                                </WalletText>
+                                {activeWalletData?.is_virtual && <WalletListCardBadge isDemo label='virtual' />}
+                            </div>
                             <WalletText color='less-prominent' size='xs'>
-                                12345678
+                                {details?.loginid}
                             </WalletText>
                         </div>
                     </div>
-                    <WalletText weight='bold'>0.00 USD</WalletText>
+                    <div className='wallets-mt5-trade-screen__details-description--right'>
+                        <WalletText weight='bold'>{details?.display_balance}</WalletText>
+                        {!activeWalletData?.is_virtual &&
```

Comment:
Created another badge `No new positions` that is only shown for real, when market type is Derived and Financial, and landing company is SVG
<img width="1808" alt="Screenshot 2023-10-26 at 3 53 39 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/37938e9f-e373-44d3-8d7c-53b09ef3ad4d">